application/src/main/java/org/thingsboard/server/exception/ThingsboardErrorResponse.java

```diff
@@ -75,7 +75,7 @@ public ThingsboardErrorCode getErrorCode() {
     }
 
     @ApiModelProperty(position = 4, value = "Timestamp", accessMode = ApiModelProperty.AccessMode.READ_ONLY)
-    public Date getTimestamp() {
+    public long getTimestamp() {
```

Comment:
Please add new Swagger schema of this class from http://localhost:8080/v3/api-docs?group=thingsboard to PR description